man/docker-build.1.md

```diff
@@ -229,16 +248,17 @@ as context. The Dockerfile at the root of the repository is used as
 Dockerfile. This only works if the GitHub repository is a dedicated
 repository.
 
-    docker build github.com/scollier/Fedora-Dockerfiles/tree/master/apache
+    docker build github.com/scollier/purpletest
```

Comment:
Install of Apache failed for me, LOL, but that's fine; the concept works :+1:

(I think we should have a repo for things like this)